data/events/2023-melbourne.yml

```
@@ -167,8 +171,8 @@ program:
   - title: "Lightning Talks"
     type: custom
     date: 2023-03-16T00:00:00+10:00
-    start_time: "13:00"
-    end_time: "13:29"
+    start_time: "13:30"
+    end_time: "13:59"
   - title: "Open Spaces"
     type: custom
```

Not a blocker, but you may wish to use the `open-space` programme type for your open spaces in a future PR. 😄 

```suggestion
  - title: "Open Spaces"
    type: open-space
```

---

Thanks Daniel, will do in my next PR!